ads/_config.js

```diff
@@ -308,12 +308,22 @@ const adConfig = jsonConfiguration({
     renderStartImplemented: true,
   },
 
+  'beop': {
+    prefetch: 'https://widget.beop.io/sdk.js',
+    preconnect: [
+      'https://t.beop.io',
+      'https://s.beop.io',
+      'https://data.beop.io',
+    ],
+    renderStartImplemented: true,
```

Comment:
I dont see this API being called in on the example page. Is it because there is no content returned from your server?